lld/test/MachO/objc-category-conflicts.s

```diff
@@ -1,3 +1,6 @@
+# TODO: `-objc_relative_method_lists` is turned on by default.
+# Adding `-no_objc_relative_method_lists` explicitly here to avoid crashing.
```

Comment:
I think we can make the situation clearer with something like the below.
Could you also file a github issue and link it in the comment ?  
```suggestion
# Adding -no_objc_relative_method_lists was needed to be explicitly added to this test to avoid it crashing after `-objc_relative_method_lists` was made default.
# TODO: Make the test compatible with default -objc_relative_method_lists and remove the -no_objc_relative_method_lists flag . Issue #FILE_AN_ISSUE
```

Reply:
Is my understanding correct that these crashes only occur when category merging (which is still of by default) is enabled alongside relative method lists?

If not, I'd like to see the crashes fixed first.

Reply:
Yes for the cases modified here.

Reply:
I updated the tests to reflect that both flags are needed to crash.